pkg/bootstrap/docker/run/run.go

```diff
@@ -112,7 +125,11 @@ func (h *Runner) Env(env ...string) *Runner {
 
 // Privileged tells Docker to run the container as privileged
 func (h *Runner) Privileged() *Runner {
```

Comment:
`Privileged` option doesn't work in user namespace mode, so I'm emulating it's behavior. Because signature of `Privileged` hasn't changed, we don't need to update existing code.